packages/jobs/lib/workflows.ts

```diff
@@ -2,18 +2,50 @@ import { CancellationScope, proxyActivities, isCancellation } from '@temporalio/
 import type * as activities from './activities.js';
 import type { WebhookArgs, ContinuousSyncArgs, InitialSyncArgs, ActionArgs } from './models/worker';
 
-const DEFAULT_TIMEOUT = '24 hours';
-const MAXIMUM_ATTEMPTS = 3;
+const SYNC_TIMEOUT = '24h';
+const SYNC_MAX_ATTEMPTS = 3;
+const ACTION_TIMEOUT = '15m';
+const ACTION_MAX_ATTEMPTS = 1; // no retry
+const WEBHOOK_TIMEOUT = '15m';
+const WEBHOOK_MAX_ATTEMPTS = 3;
```

Comment:
imo all constants like should always be configurable via process.env

Reply:
we need to centralize our usage of env var that's for sure, it was just a general statement over this pattern